internal/convertor/device/device.go

```diff
@@ -63,7 +69,7 @@ func NewDevice(dcimInfo *dcim.NetworkDevice, devicesData *repository.AssetsPerDe
 	// Check if there is CMDB data for the device
 	device.Sessions, ok = devicesData.BGPsessions[dcimInfo.Hostname]
 	if !ok {
-		return nil, fmt.Errorf("no BGP session found for %s", dcimInfo.Hostname)
```

Comment:
please do not do that. Return an error is done on purpose here.
The only exception is for BGP Global, for which there is a comment.

Reply:
ok